backend/app/adapter/db/change_log_integration_test.go

```diff
@@ -43,23 +44,23 @@ func TestChangeLogSql_GetChangeLog(t *testing.T) {
 				{
 					id:              "12346",
 					title:           "title 2",
-					summaryMarkdown: "summary 2",
+					summaryMarkdown: summaryMarkdown,
```

Comment:
Let's put explicitly write out `summaryMarkdown` for each test case. Let's also add a case for `nil` summaryMarkdown.

Reply:
I have already added a case for nil summaryMarkdown